pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -165,6 +165,24 @@ func (c *cephStatusChecker) updateCephStatus(status *cephclient.CephStatus, cond
 	// Update with Ceph Status
 	cephCluster.Status.CephStatus = toCustomResourceStatus(cephCluster.Status, status)
 
+	// versions store the ceph version of all the ceph daemons and overall cluster version
+	versions, err := cephclient.GetAllCephDaemonVersions(c.context, c.clusterInfo)
+	if err != nil {
+		logger.Errorf("failed to get ceph daemons versions; this likely means there is no cluster yet. %v", err)
```

Comment:
```suggestion
		logger.Errorf("failed to get ceph daemons versions. %v", err)
```

Reply:
`this likely means there is no cluster yet` should not be the case here as we are already checking that in line 155 above.

Reply:
done.